src/processing/vFramer/include/vDraw.h

```diff
@@ -51,10 +51,11 @@ class vDraw {
     unsigned int display_window;
     unsigned int max_window;
     bool flip;
+    bool scaling;
 
 public:
 
-    vDraw() : Xlimit(304), Ylimit(240), flip(false)
+    vDraw() : Xlimit(304), Ylimit(240), flip(true)
```

Comment:
don't change the default options of the base class vDraw - it will affect all other drawers too. The defualt is FALSE